### DISCUSSION TITLE
Can't make it to work

### DISCUSSION THREAD

**Discussion by americodias** [paperboi/kindle2notion#83] [Q&A]
Hello everyone,

I've tried to install kindle2notion but wasn't able to make it work. I can't find this step of the installation manual:

"Select the Integrations option listed under Workspaces in the settings modal."

I went to https://www.notion.so/my-integrations tough and created an integration. But then again when I go to the database page to the share option I don't find my integration there.

Maybe the notion UI has changed? Any help is appreciated.

Regards,
Américo

**Comment by paperboi** [comment]
Hi, please refer to Step 2 [here](https://developers.notion.com/docs/create-a-notion-integration) on how to connect your token to your database. The setup process has slightly changed and I'll add the necessary updates to the README later this week. Thanks for bringing this up!

**Comment by americodias** [comment]
Hi again,

I tried to follow the example you mention. I have shared the integration with the database page:

<img width="472" alt="Screenshot 2022-12-07 at 20 47 12" src="https://user-images.githubusercontent.com/4559733/206281258-e3a94740-efb9-4f2c-b464-8072fba72057.png">

Still, when I run their kindle2notion I get:

> requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://www.notion.so/api/v3/loadUserContent

Thanks,
Américo

**Reply by paperboi** [reply]
> Hi again,
> 
> I tried to follow the example you mention. I have shared the integration with the database page:
> 
> <img alt="Screenshot 2022-12-07 at 20 47 12" width="472" src="https://user-images.githubusercontent.com/4559733/206281258-e3a94740-efb9-4f2c-b464-8072fba72057.png">
> 
> Still, when I run their kindle2notion I get:
> 
> > requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: [notion.so/api/v3/loadUserContent](https://www.notion.so/api/v3/loadUserContent)
> 
> Thanks, Américo

Please try logging in again on the browser you're on and run the command. 401 means it's a client side error which happens occasionally. Usually logging out and in again  or  clearing the browser cache does the trick for this.

**Comment by pdbg** [comment]
@paperboi It's a great project! Thanks for your contribution!

I followed the steps mentioned in the main page of the repository and steps narrated here in this discussion. I don't see access denied or unable to find database kind of errors. However, getting below mentioned errors:

![image](https://user-images.githubusercontent.com/6949452/209402299-98ee00f5-8b6a-4b8b-929e-4de99a812420.png)

Would you please direct me to solve the issue?

**Reply by pdbg** [reply]
@paperboi I've got the clue why it was breaking and I've fixed the same on my fork and raised PR as well - https://github.com/paperboi/kindle2notion/pull/86.

**Reply by paperboi** [reply]
Thanks @pdbg! I'll take a look at it in the next few days and merge it.
